taichi/backends/vulkan/runtime.h

```diff
@@ -46,6 +48,10 @@ class VkRuntime {
 
   void synchronize();
 
+#ifdef TI_WITH_VULKAN
```

Comment:
Shouldn't really need to guard this: `VkRuntime` is implemented using the C++ Pimpl pattern.